src/rpc/misc.cpp

```diff
@@ -337,6 +338,33 @@ static RPCHelpMan verifymessage()
     };
 }
 
+static RPCHelpMan savefeeestimates()
+{
+    return RPCHelpMan{"savefeeestimates",
+                "\nDumps the fee estimates to disk. It will fail until the previous dump is fully loaded.\n",
+                {},
+                RPCResult{RPCResult::Type::NONE, "", ""},
+                RPCExamples{
+                    HelpExampleCli("savefeeestimates", "")
+            + HelpExampleRpc("savefeeestimates", "")
+                },
+        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
+    static Mutex dump_mutex;
+    LOCK(dump_mutex);
+    CBlockPolicyEstimator& fee_estimator = EnsureAnyFeeEstimator(request.context);
+
+    fs::path est_filepath = gArgs.GetDataDirNet() / CBlockPolicyEstimator::FEE_ESTIMATES_FILENAME;
+    CAutoFile est_file(fsbridge::fopen(est_filepath, "wb"), SER_DISK, CLIENT_VERSION);
+    if (est_file.IsNull() || !fee_estimator.Write(est_file)) {
```

Comment:
IMO better to abstract this to fees.cpp

Reply:
Done, I called the method Write, without parameters - does it make sense to you to overload the method?

Reply:
No opinion on that